native/src/components/PoiFiltersModal.tsx

```diff
@@ -110,17 +119,22 @@ const PoiFiltersModal = ({
             <SubTitle>{t('poiCategories')}</SubTitle>
             <SortingHint>{t('alphabetLetters')}</SortingHint>
           </Row>
-          <TileRow>
-            {poiCategories.map(it => (
+          <FlatList
+            data={poiCategories}
+            numColumns={numOfColumns}
+            style={{ marginVertical: 10, height: flatListHeight }}
+            contentContainerStyle={{ gap: 16, alignItems: 'center' }}
+            columnWrapperStyle={{ gap: 16 }}
+            renderItem={({ item }) => (
```

Comment:
Very nice, thank you for making the code more humanly readable by changing `it` to `item`!